proto/snpb/log_stream_reporter.go

```diff
@@ -8,6 +8,12 @@ import (
 var InvalidLogStreamUncommitReport = LogStreamUncommitReport{}
 var InvalidLogStreamCommitResult = LogStreamCommitResult{}
 
+func (m *LogStreamUncommitReport) Invalid() bool {
+	// uncommitReport with invalid logStream or invalid uncommittedLLSNOffset
+	// is not acceptable. MetadataRepository ignores these reports.
```

Comment:
Could you move this to comment for the method?
Please see Funcs at https://go.dev/doc/comment.

Reply:
Okay